lib/ex_subtil_backend/workflow/step/requirements.ex

```diff
@@ -0,0 +1,27 @@
+defmodule ExSubtilBackend.Workflow.Step.Requirements do
+
+  def get_path_exists(requirements \\ %{}, _path)
```

Comment:
Rename with `get_required_paths`
Using exists can mean a check from the backend